### PR TITLE
Cwa 282 ios seed backup card

### PR DESCRIPTION
**Pull request by Jemma-code**

### CODE REVIEW

**Comment by Jemma-code** [comment]
@skchang0710  @dannyyaoAtCoolbitx plz code review~

**Comment by dannyyaoAtCoolbitx** [comment]
LGTM!

**Review by dannyyaoAtCoolbitx — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,123 @@
...
+> SeedBackupCard SampleCode for CoolWalletS
+
+```
+version:    1.0.0
+status:     release
+copyright:  coolbitX
+```
+
+## Support Devices
+- iPhone 7 / iPhone 7 Plus
```
Comment (skchang0710) [inline README.md]: Should adding "and later" at the end of this line be more precise ?
https://english.stackexchange.com/questions/112257/above-or-later-when-referencing-a-range-of-versions-of-software

Comment (Jemma-code) [reply README.md]: @skchang0710 resolve!

**Review by skchang0710 — APPROVED** [review]

**Pull request merged**